packages/atlas/src/hooks/useNftTransactions.ts

```diff
@@ -62,7 +62,22 @@ export const useNftTransactions = () => {
     [_refetchData, activeMemberId, closeModal, handleTransaction, joystream, openModal, proxyCallback]
   )
 
+  const changeNftPrice = useCallback(
+    (id: string, price: number) => {
+      if (!joystream || !activeMemberId) {
+        return
+      }
+      handleTransaction({
```

Comment:
There should be a snackbar once finished: https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=1247%3A107844

Like here:
https://github.com/Joystream/atlas/blob/a5c0761ec86c655db2f3234272695f3f5ae75009/packages/atlas/src/views/global/NftSaleBottomDrawer/NftSaleBottomDrawer.tsx#L53-L73